airdnb/src/main/java/team10/airdnb/accommodation_amenity/service/AccommodationAmenityService.java

```diff
@@ -0,0 +1,43 @@
+package team10.airdnb.accommodation_amenity.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import team10.airdnb.accommodation.entity.Accommodation;
+import team10.airdnb.accommodation_amenity.dto.AccommodationAmenityDto;
+import team10.airdnb.accommodation_amenity.entity.AccommodationAmenity;
+import team10.airdnb.accommodation_amenity.repository.AccommodationAmenityRepository;
+import team10.airdnb.amenity.entity.Amenity;
+import team10.airdnb.amenity.repository.AmenityRepository;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class AccommodationAmenityService {
+
+    private final AccommodationAmenityRepository accommodationAmenityRepository;
+    private final AmenityRepository amenityRepository;
+
+    public AccommodationAmenityDto saveAccommodationAmenity(Accommodation savedAccommodation, List<Long> amenityIds) {
+        List<Amenity> amenities = amenityRepository.findAllById(amenityIds != null ? amenityIds : Collections.emptyList());
```

Comment:
```suggestion
        if (amenityIds == null) return Collections.emptySet();
        List<Amenity> amenities = amenityRepository.findAllById(amenityIds);
```

지금 작성하신 코드 처럼 하셔도 좋고 아니면 위 코드 처럼 amenityIds 가 empty 일땐 별도로 실행될 이유가 없어 보여서 위에서 null 일때 early return 해주는 방향도 좋은거 같아요.

Reply:
아하 그런 방법도 있군요, 삼항 연산자를 사용해서 null인지 아닌지를 검증하는 것보다 차라리 null 일때 early return을 하여서 필요없는 검증 로직을 줄일수도 있겠네요! 감사합니다!!